scripts/update-config/index.ts

```diff
@@ -56,12 +58,9 @@ async function main() {
 }
 
 function dtConfig(updatedRules: string[]): Config.IConfigurationFile {
-    const resolvedDtslint = require.resolve('dtslint');
+    const resolvedDtslint = require.resolve("dtslint");
```

Comment:
I'm a little confused by the quote styles used in this PR, some files change to single quotes, while some change to double quotes. Is it just to be consistent within the file? There are other formatting changes that would imply `prettier` or something, but if so, we should end up with consistent quoting.